src/boot/stylesheets/actions.scss

```diff
@@ -57,7 +57,7 @@
 	}
 }
 
-.wpnc__action-link {
+.wpnc__main .wpnc__action-link {
```

Comment:
The general resets do need to go under `.wpnc__main` otherwise Calypso will pick up those rules as well. 😄 

Depending on the order of declaration `button.wpnc__action-link` might also work, but this is fine here too.